### PR TITLE
Timer: Fixing trigger

### LINKED ISSUE #4038: Timer: Improve triggering
## Problem

Based on user feedback and unanswered query data, This IA isn't triggering enough. @moollaza just merged a PR that ported this IA from a Spice to a Goodie.

**Triggering** 👍 

screen shot 2017-03-24 at 6 55 21 pm

**Not triggering** 👎 

screen shot 2017-03-24 at 6 55 38 pm

## Solution

1. Add one, two, or three tests! 😉 
2. Add the trigger words `mins`, `minutes`

## People to notify

@pjhampton

## Get Started
- [ ] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/timer

### PR DESCRIPTION
**Pull request by TheSpeedyBlur**
## Description of new Instant Answer, or changes

Added trigger words of `hrs, mins, secs, hours, minutes, seconds`.
Removed one unnecessary test, which should trigger the IA.

## Related Issues and Discussions

Fixes #4038 

## People to notify

@pjhampton 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [x] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [x] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [x] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [x] There is no unnecessary files in place (such as editor config files)
- [x] There is no API keys / secrets present
- [x] Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/timer

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Timer.pm [hunk 1]
```diff
@@ -7,15 +7,17 @@ use DDG::Goodie;
...
 # Triggers that are valid in start only
 my @startTringgers = qw(start begin set run);
 # Beautifies the trigger can be appended in front/back of trigger
 my @beautifierTringgers = qw(online);
 #Joins the Timer Value
 my @joiners = qw(for on at with);
-# StartEndTriggers to trigger on startTringgers, beautifierTringgers and triggers
-my @triggersStartEnd = (@triggers, @startTringgers, @beautifierTringgers);
+# StartEndTriggers to trigger on nonStrippedTriggers, startTringgers, beautifierTringgers and triggers
+my @triggersStartEnd = (@triggers, @nonStrippedTriggers, @startTringgers, @beautifierTringgers);
```
Comment (moollaza) [inline lib/DDG/Goodie/Timer.pm]: @TheSpeedyBlur mind fixing the pre-existing typo in the code?

`startTringgers` => `startTriggers`

Comment (TheSpeedyBlur) [inline lib/DDG/Goodie/Timer.pm]: Sure, one sec.

**Comment by moollaza** [comment]
@TheSpeedyBlur Thanks! This LGTM :+1:

Excellent work 💯

**Pull request merged**